docs/en/index.md

```diff
@@ -40,6 +40,22 @@ Alternatively, you can add Vanilla directly to your markup:
 
 Download the latest version of Vanilla from [GitHub](https://github.com/vanilla-framework/vanilla-framework/releases/)
 
+### Using Git submodules
```

Comment:
Could we title this something about GitHub Pages? Since the only reason for doing this is to facilitate the simplest deployment flow with GitHub Pages.